internal/spec/metadata.go

```diff
@@ -24,6 +25,11 @@ func (m *Metadata) FromGithubOrg(o *github.Organization) {
 	m.Name = o.GetName()
 }
 
+func (m *Metadata) FromGithubTeam(t *github.Team) {
```

Comment:
@OrlovM I think we need to put `t.Slug` to metadata too, e.g. to metadata `Name`